sdk/python/foundation-models/huggingface/inference/stable_diffusion/text-to-image-online-endpoint.ipynb

```diff
@@ -0,0 +1,297 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Text to Image Inference using Online Endpoints\n",
+    "\n",
+    "This sample shows how deploy `text-to-image` type stable diffusion models to an online endpoint for inference.\n",
+    "\n",
+    "### Task\n",
+    "`text-to-image` tasks generates image at output based on text prompt given in input.\n",
+    " \n",
+    "### Model\n",
+    "Models that can perform the `text-to-image` task are tagged with `text-to-image`. We will use the `runwayml/stable-diffusion-v1-5` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name. If you don't find a model that suits your scenario or domain, you can discover and [import models from HuggingFace hub](../../../system/import/import_model_into_registry.ipynb) and then use them for inference.\n",
```

Comment:
```suggestion
    "Models that can perform the `text-to-image` task are tagged with `text-to-image`. We will use the `runwayml-stable-diffusion-v1-5` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name. If you don't find a model that suits your scenario or domain, you can discover and [import models from HuggingFace hub](../../../system/import/import_model_into_registry.ipynb) and then use them for inference.\n",
```

We will not have "/" in the model name in registry. That will be replaced by "-"